scripts/summarize_profiling.py

```diff
@@ -38,7 +38,7 @@
 )
 
 table = np.genfromtxt(p.stdout, dtype=None, names=True, comments='--', encoding=None)
-table['MaxRSS'] = map(lambda x: float(x[:-1]) * 1e3 / 1e9, table['MaxRSS'])
+table['MaxRSS'] = (float(x[:-1]) * 1e3 / 1e9 for x in table['MaxRSS'])
```

Comment:
@aelanman or @mkolopanis can someone verify that this is correct? Since this is in a script not in a tested module and I don't really have a way to test it, I'd like to make sure I don't make a mistake.

This change is because of a new check in flake8-comprehensions, see here: https://github.com/adamchainz/flake8-comprehensions/pull/409